commit 4d885764f41be161c907cb8049d79cf0d6558981 (joyent/1.10.2, refs/changes/54/5454/3)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-01-31T17:30:36+00:00 (8 months ago)
    
    MANTA-4093 nginx should not modify its source tree during the build
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 00000000..9544294d
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,2 @@
+/Makefile
+/objs
